src/org/labkey/test/components/ui/ontology/OntologyTreeSearch.java

```diff
@@ -55,12 +58,76 @@ public OntologyTreeSearch selectItemWithLabel(String searchExpression, String la
         return this;
     }
 
-    private OntologyTreeSearch setInput(String value)
+    public boolean showsNoSearchResultsFound()
     {
+        return Locator.tagWithClass("div", "col").withText("No search results found.")
+                .existsIn(elementCache().resultContainer());
+    }
+
+    public boolean isResultContainerExpanded()
+    {
+        try
+        {
+            return elementCache().resultContainer().isDisplayed();
+        } catch (NoSuchElementException nse)
+        {
+            return false;
+        }
+    }
+
+    public OntologyTreeSearch setInput(String value)
+    {
+        clearInput();
         elementCache().searchInput.set(value);
         return this;
     }
 
+    /**
+     * sets the input text to empty and waits for the search results container to collapse
+     * and for the placeholder to become visible
+     * @return
+     */
+    public OntologyTreeSearch clearInput()
+    {
+        var input = elementCache().searchInput;
+        String placeholder = input.getComponentElement().getAttribute("placeholder");
+        input.set("");
+        // wait for the results container to collapse, and for the placeholder for the input to be shown
+        getWrapper().waitFor(()-> !isResultContainerExpanded() &&
+                Locator.css("input:placeholder-shown").withAttribute("placeholder", placeholder)
+                        .existsIn(this),  2000);
+        return this;
+    }
+
```

Comment:
Because I wanted to write tests that would search on a repeated basis, I needed a way to wait for search results to appear before returning them.  This also called for a way to clear those results, so I could begin from a state in which there were no searchResults to find.